--- mdk-1.2.4/mixguile/Makefile.am.orig	2008-03-20 14:44:32.000000000 +0100
+++ mdk-1.2.4/mixguile/Makefile.am	2008-03-22 19:49:00.082002957 +0100
@@ -31,7 +31,7 @@
 bin_PROGRAMS = mixguile
 mixguile_LDADD = $(top_builddir)/mixlib/libmix.a \
                  $(top_builddir)/lib/libreplace.a \
-                 $(top_builddir)/mixguile/libmixguile.a $(INTLLIBS)
+                 libmixguile.a $(INTLLIBS)
 mixguile_SOURCES = mixguile_main.c
 
 else
